plenum/server/node.py

```diff
@@ -2108,6 +2108,12 @@ def allLedgersCaughtUp(self):
             logger.info('{}{} caught up till {}'
                         .format(CATCH_UP_PREFIX, self, last_caught_up_3PC),
                         extra={'cli': True})
+        to_pop = []
+        for key, r in self.requests.items():
+            if not r.forwarded and self.seqNoDB.get(key)[1]:
+                to_pop.append(key)
+        for key in to_pop:
+            self.requests.pop(key, None)
```

Comment:
This is not enough to clear the request.
The following must be called in addition:
```
self._clean_req_from_verified(req_state.request)
self.doneProcessingReq(req_key)
```

Reply:
done